sigstore-java/src/main/java/dev/sigstore/KeylessVerifier.java

```diff
@@ -72,9 +77,14 @@ public KeylessVerifier build()
       Preconditions.checkNotNull(trustedRootProvider);
       var trustedRoot = trustedRootProvider.get();
       var fulcioVerifier = FulcioVerifier.newFulcioVerifier(trustedRoot);
-      var rekorClient = RekorClient.builder().setTransparencyLog(trustedRoot).build();
       var rekorVerifier = RekorVerifier.newRekorVerifier(trustedRoot);
-      return new KeylessVerifier(fulcioVerifier, rekorClient, rekorVerifier);
+      var rekorClients =
+          trustedRoot.getTLogs().getTransparencyLogs().stream()
+              .map(TransparencyLog::getBaseUrl)
+              .distinct()
+              .map(uri -> RekorClient.builder().setUri(uri).build())
+              .collect(Collectors.toList());
```

Comment:
Does this preserve order so the newer logs are first? This might help when it comes to searching all logs (unless you're mostly verifying old artifacts).